dags/js/aws/get-s3-client.js

```diff
@@ -0,0 +1,22 @@
+const AWS = require('aws-sdk');
+
+function getS3Client() {
+
+  let s3ConfigParams = {
+    apiVersion: '2006-03-01',
+    maxRetries: 3,
+    s3ForcePathStyle: true,
```

Comment:
@giorgiosironi assumed this would be host style, so he will investigate.

Reply:
I would expect that this is required by MinIO rather than S3, as AWS is deprecating path style:
https://aws.amazon.com/blogs/aws/amazon-s3-path-deprecation-plan-the-rest-of-the-story/

Reply:
Correction: would expect this is required for all fake implementations of S3.

Reply:
@GiancarloFusiello to verify `s3ForcePathStyle: false` would work with a real S3; use local tests and personal credentials to do it